README.md

```diff
@@ -0,0 +1,43 @@
+# Layer0 Log Forwarder
+
+This terraform module forwards logs generated by services and tasks deployed via Layer0 to a Kinesis stream.
+
+## Overview
+
+Each layer0 instance create a Cloudwatch Logs Group named `l0-<layer0_instance_prefix>`.
+
+This module will create infrastructure to forward all logs from a log group to a Kinesis Stream. Extra Layer0 metadata will be added to each log event, to map a given event to an Layer0 instance, environment and service.
+
+For Example:
+
+##### Cloudwatch log event
+```
+[Warning] Running out of disk space
+```
+
+##### What gets written to the Kinesis stream
+```json
+{
+    "Severity": "Warning",
+    "Message": "[WARNING] Running out of disk space",
```

Comment:
how does this fit in with apps that already have an existing framework-specific format, ie ImsHealth.Logging?

Reply:
This structure mimics what is output by ImsHealth.Logging. So the intention is that the user of this module would forward to one of our logging pipeline streams and a lambda forwarder would be able to pick it up and forward to the hec endpoints.

If you search in splunk for `host=layer0` you will see some of the data that has been sent via this module.